.gitignore

```diff
@@ -6,3 +6,4 @@
 .sass-cache
 node_modules/
 test/
+package-lock.json
```

Comment:
I could but it would be redundant. Just don't run the `npm shrinkwrap` command

Reply:
> Although I'm not fully aware of when npm-shrinkwrap.json is created, this file was created and committed by npm publish. In this case, I think adding it to .gitignore would be essential.

This will also prevent any future problem/accident/misunderstanding to happen.

Reply:
Ok, no problem. Added